docusaurus/docs/Android/03-ui/02-general-customization/02-chatui.mdx

```diff
@@ -289,3 +469,33 @@ ChatUI.dateFormatter = object: DateFormatter {
     }
 }
 ```
+</TabItem>
+
+<TabItem value="java" label="Java">
+
+```java
+ChatUI.setDateFormatter(new DateFormatter() {
```

Comment:
```suggestion
ChatUI.INSTANCE.setDateFormatter(new DateFormatter() {
```

Reply:
we don't need to use `INSTANCE` here as `dateFormatter` is marked with @JvmStatic